src/main/fc/fc_core.c

```diff
@@ -389,13 +402,61 @@ static bool canUpdateVTX(void)
 }
 #endif
 
+// determine if the R/P/Y stick deflection exceeds the specified limit - integer math is good enough here.
+uint8_t areSticksActive(uint8_t stickPercentLimit) {
```

Comment:
```
bool areSticksActive(uint8_t stickPercentLimit)
{
```

Reply:
Yeah, this was a mistake leftover from earlier revisions.  Originally I had a function that returns the stick percent for a supplied axis and the calling code would compare the limit.  Simplified to make one call and that checked all axis and returned boolean but forgot to update the return data type.  I'll fix.